modules/db/src/main/scala/gem/dao/DatasetDao.scala

```diff
@@ -31,7 +31,7 @@ object DatasetDao {
     private final case class StepId(toInt: Int)
     private object StepId {
       implicit val StepIdMeta: Meta[StepId] =
-        Distinct.integer("id_index").xmap(StepId(_), _.toInt)
+        Distinct.integer("id_index").timap(StepId(_))(_.toInt)
```

Comment:
`timap`?

Reply:
It's a doobie thing. Identical to `imap` but asks for a type tag for `B`, which doobie can use in error messages.